app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::Admin::UsersController < Api::V1::Admin::BaseController
+  before_action :authorize_access_request!, :require_admin
+  
```

Comment:
Right now admin has access to index and show for all the users in the database. If the admin wished to update their profile, they can access it through the users controller